pkg/commands/objectstorekeys/insert.go

```diff
@@ -0,0 +1,45 @@
+package objectstorekeys
```

Comment:
For consistency we should name this `create.go`

Reply:
Done.